### DISCUSSION TITLE
500 - Server Error

### DISCUSSION THREAD

**Discussion by sliine** [Kovah/LinkAce#602] [Support] [answered]
Hello, I created docker-conpose and change .env, give 666 .env and entered this errors -**An internal server error occured. If you are the administrator, consult the application logs for details.**. Everything you've written in other posts doesn't help. How fix?

**Comment by Kovah** [comment, marked as answer]
Please add the following line to your env file and try again, then post the exact error messages.

`APP_DEBUG=true`

**Reply by sliine** [reply]
https://imgur.com/a/dDqhlMZ - this error messages.

**Reply by Kovah** [reply]
Which Docker setup did you choose? The simple one? Try to delete the logs volume and try again.

```
docker compose down
docker volume rm linkace_logs
docker compose up -d
```

**Reply by sliine** [reply]
> Какую настройку Docker вы выбрали? Простой? Попробуйте удалить том журналов и повторите попытку.
> 
> ```
> docker compose down
> docker volume rm linkace_logs
> docker compose up -d
> ```

Thanks, i give 777 permissions and worked/

**Reply by HassanMullah** [reply]
Yes, that's how it was for me too. The instructions say 666, but you should first do 777, then set everything and then go back.

**Comment by driftywinds** [comment]
This helped me in July 2024 too! Thanks alot!
